frontend/src/gen/api.ts

```diff
@@ -117,17 +118,23 @@ export interface ModelClass {
      */
     rating: number;
     /**
-     * 
-     * @type {string}
+     * 0:spring 1:autumn 2:other
+     * @type {number}
      * @memberof ModelClass
```

Comment:
実行しようとしたら```This condition will always return 'false' since the types 'number' and 'string' have no overlap.```となってエラー吐かれるんだけどそっちの環境でも起こる??

Reply:
ERROR in /Users/kanade/Git/pj-marowd/frontend/src/templates/DetailPage.vue(68,9):
68:9 This condition will always return 'false' since the types 'number' and 'string' have no overlap.  
>
    66 |
    67 |   get termColor(): string {
    68 |     if (this.modelClass.term === 'spring') {
       |         ^
    69 |       return '#ffebee';
    70 |     }
    71 |     if (this.modelClass.term === 'autumn') {

Reply:
ERROR in /Users/kanade/Git/pj-marowd/frontend/src/templates/DetailPage.vue(71,9):
71:9 This condition will always return 'false' since the types 'number' and 'string' have no overlap.
>
    69 |       return '#ffebee';
    70 |     }
    71 |     if (this.modelClass.term === 'autumn') {
       |         ^
    72 |       return '#FFF3E0';
    73 |     }
    74 |     return '#ECEFF1';

Reply:
ERROR in /Users/kanade/Git/pj-marowd/frontend/src/views/NewOrEditPage.vue(97,7):
Type 'string' is not assignable to type 'number'.
>
    95 |       isRandom: this.lottery,
    96 |       rating: 0.0,
    97 |       term: this.term,
        |       ^
     98 |       editedBy: user.displayName as string
     99 |     };
    100 |

Reply:
@kanade9 
その辺、まだ、改修していない所なので見なくて大丈夫です！

DetailPage.vueは詳細画面のタスク時で直すし、NewOrEditPage.vueは新規作成・編集画面のタスクで直す心持ち。

Reply:
OKです!!